test-support/page-object/properties/click-on-text.js

```diff
@@ -45,11 +51,22 @@ export function clickOnText(selector, options = {}) {
     isDescriptor: true,
 
     value(textToClick) {
-      wait().then(() => {
-        var actualSelector = findChildElement(this, selector, textToClick, options) || findElement(this, selector, textToClick, options);
+      var context = getContext(this);
+
+      if (context) {
+        Ember.run(() => {
+          var fullSelector = actualSelector(this, selector, textToClick, options);
+
+          context.$(fullSelector).click();
+        });
+      } else {
```

Comment:
Looking at it more closely I think the issue is that you wrote wait with back ticks. Try removing the back ticks.